di-ipv-ticf-stub/lambdas/src/services/ticfService.ts

```
@@ -0,0 +1,120 @@
+import { JWTPayload } from "jose";
+import { v4 as uuid } from "uuid";
+
+import { buildSignedJwt } from "stub-oauth-client";
+import type { SignedJwtParams } from "stub-oauth-client";
+
+import { getSsmParameter } from "../common/ssmParameter";
+import TicfRequest from "../domain/ticfRequest";
+import TicfResponse from "../domain/ticfResponse";
+
+export class TicfService {
+  async processGetVCRequest(ticfRequest: TicfRequest): Promise<TicfResponse> {
+    // preparing response
+    let ticfSigningKey: string;
+    try {
+      ticfSigningKey = await getSsmParameter(
+        process.env.TICF_PARAM_BASE_PATH + "signingKey"
+      );
+    } catch (error) {
+      throw new Error(`Error while retrieving TicF CRI VC signing key.`);
+    }
```

This is swallowing the underlying error - can we just let that bubble up? An error message describing the parameter being fetched feels more useful anyway.

---

done